Extensions/keyboardShortcut.js

```diff
@@ -1,236 +1,102 @@
 // NAME: Keyboard Shortcut
-// AUTHOR: khanhas
+// AUTHOR: khanhas, OhItsTom
 // DESCRIPTION: Register a few more keybinds to support keyboard-driven navigation in Spotify client.
 
 /// <reference path="../globals.d.ts" />
 
 (function KeyboardShortcut() {
-	if (!Spicetify.Keyboard) {
+	if (!Spicetify.Mousetrap) {
 		setTimeout(KeyboardShortcut, 1000);
 		return;
 	}
 
+	// Variables / Conditions
+	const vim = new VimBind();
 	const SCROLL_STEP = 25;
 
 	/**
-	 * Register your own keybind with function `registerBind`
-	 *
-	 * Syntax:
-	 *     registerBind(keyName, ctrl, shift, alt, callback)
-	 *
-	 * ctrl, shift and alt are boolean, true or false
-	 *
-	 * Valid keyName:
-	 * - BACKSPACE       - C               - Y               - F3
-	 * - TAB             - D               - Z               - F4
-	 * - ENTER           - E               - WINDOW_LEFT     - F5
-	 * - SHIFT           - F               - WINDOW_RIGHT    - F6
-	 * - CTRL            - G               - SELECT          - F7
-	 * - ALT             - H               - NUMPAD_0        - F8
-	 * - PAUSE/BREAK     - I               - NUMPAD_1        - F9
-	 * - CAPS            - J               - NUMPAD_2        - F10
-	 * - ESCAPE          - K               - NUMPAD_3        - F11
-	 * - SPACE           - L               - NUMPAD_4        - F12
-	 * - PAGE_UP         - M               - NUMPAD_5        - NUM_LOCK
-	 * - PAGE_DOWN       - N               - NUMPAD_6        - SCROLL_LOCK
-	 * - END             - O               - NUMPAD_7        - ;
-	 * - HOME            - P               - NUMPAD_8        - =
-	 * - ARROW_LEFT      - Q               - NUMPAD_9        - ,
-	 * - ARROW_UP        - R               - MULTIPLY        - -
-	 * - ARROW_RIGHT     - S               - ADD             - /
-	 * - ARROW_DOWN      - T               - SUBTRACT        - `
-	 * - INSERT          - U               - DECIMAL_POINT   - [
-	 * - DELETE          - V               - DIVIDE          - \
-	 * - A               - W               - F1              - ]
-	 * - B               - X               - F2              - "
-	 *
-	 * Use one of keyName as a string. If key that you want isn't in that list,
-	 * you can also put its keycode number in keyName as a number.
-	 *
-	 * callback is name of function you want your shortcut to bind to. It also
-	 * returns one KeyboardEvent parameter.
-	 *
-	 * Following are my default keybinds, use them as examples.
+	 * Binds a keyboard shortcut using Mousetrap.
+	 * @param {string} key - The Mousetrap keybind.
+	 * @param {boolean | undefined} staticCondition - A static condition.
+	 * @param {(event: KeyboardEvent) => void} callback - Callback function for the event.
 	 */
-
-	// Ctrl + Tab and Ctrl + Shift + Tab to switch sidebar items
-	registerBind("TAB", true, false, false, rotateSidebarDown);
-	registerBind("TAB", true, true, false, rotateSidebarUp);
-
-	// Ctrl + Q to open Queue page
-	registerBind("Q", true, false, false, clickQueueButton);
```

Comment:
Why did this one not get ported?

Reply:
Apparently it's Alt+Shift+q; although I liked many of the previous bindings better, as the required less keypresses. Could be nice to be able to customize the bindings, even for those implemented by Spotify themselves.

Reply:
If you make an issue, provided others agree im happy to reimplement, however we shouldnt really have duplicate keybindings for the same function.